test/factories/compute_resources.rb

```diff
@@ -49,7 +49,6 @@
       provider 'Vmware'
       user 'vuser'
       password 'vpassword'
-      sequence(:url) { |n| "#{n}.example.com" } # alias for server
```

Comment:
Why is this necessary? It looks like a good default to me. You cannot use `sequence(:server)` as FactoryGirl doesn't allow you to use an aliased method.